coda/templates/mdstore/bag_files_download.html

```diff
@@ -0,0 +1,15 @@
+{% extends "mdstore/base.html" %}
+{% block content %}
+{% load humanize %}
+<table>
+    {%for item in outputText%}
+        <tr>
+            <td>
+                <a href="item" download>{{ item }}</a>
+            </td>
+        </tr>
+    {% endfor %}
+</table>
+</body>
+</html>
```

Comment:
I think we could simplify this by leaving out the `<table>` and also we don't need to include the `</body>` and `</html>` tags, since they will be included by the `endblock`. Also, currently the links are not working. Perhaps we could use:
```suggestion
{% for item in outputText %}
    <a href="{{ item }}">{{ item }}</a><br>
{% endfor %}
```

Reply:
Yeah...I'll have to change that to `<a href="{{ item }}">`. I was thinking we'd want to have downloadable links here. The way we have now along with the above mentioned change is, on clicking the link it loads the web page, but not download those files.